### ISSUE TITLE
Feature Request: Alternative options for Verproc into jolt plus

### ISSUE THREAD

**Issue by 0kie** [Grammernatzi/XIVComboPlugin#165]
Hello.
Currently the Verproc into Jolt plus option works with Verfire becoming Jolt when you dont have a proc and Verthunder when you have an instant (and the same for Verstone).

Would be possible to add an option so that Verthunder becomes jolt when you dont have an instant and Verfire when you dont have an instant and have the proc and is not in the level 84~93 range (and the same for Veraero) instead of Verfire and Verstone changing?

and also keep the opener features, so out of combat they remain Verthunder/Veraero

In short I want everything to stay the same as current "Verproc into jolt plus" but an option so the action that changes into others to be Verthunder/Veraero instead of Verfire/Verstone

This would solve the opener incompatibility by allowing us to separately bind verfire and/or verstone to a separated button

**Comment by Grammernatzi** [comment]
I'm looking at the opener incompatibility right now and that is some janky
stuff happening at the end of the embolden period. I will look into an
option that lets you change Verthunder as a base instead of Verfire to get
around that.

**Comment by Grammernatzi** [comment]
Added, should be good now.
